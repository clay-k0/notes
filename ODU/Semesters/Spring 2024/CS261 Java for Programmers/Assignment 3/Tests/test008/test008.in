13
A 1
B 2
C 3 A B
D 5 C B
E 8 D C
F 13 E D
G 21 E F
H 34 F G
I 55 G H
J 89 H I
K 1
L 2
M 3 K L
N 5 L M
O 8 M N
P 13 N O
Q 21 O P
R 34 P Q
S 55 R Q
T 89 S R
U 1
V 2
W 3 U V
X 5 W V
Y 8 X W
Z 13 Y X
