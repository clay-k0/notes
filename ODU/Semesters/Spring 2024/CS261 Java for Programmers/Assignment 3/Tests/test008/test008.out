237:  A:1  B:2  C:3  D:5  E:8  F:13  G:21  H:34  I:55  J:89  K:1  L:2  M:3
